Src/coffeescript/cql-execution/src/example/browser/cql4browsers.js

```diff
@@ -3626,7 +3626,7 @@
       ref = this.element;
       for (i = 0, len = ref.length; i < len; i++) {
         el = ref[i];
-        obj[el.name] = el.execute(ctx);
+        obj[el.name] = el.exec(ctx);
```

Comment:
please add similar comments to these changes for why execute should not be called.